AirCasting/SessionViews/SessionCartView.swift

```diff
@@ -107,7 +128,13 @@ private extension SessionCartView {
     }
     
     func displayButtons(threshold: SensorThreshold) -> some View {
-        HStack(spacing: 20){
+        HStack(spacing: 20) {
+            if isFollowing && !(session.type == .mobile) {
+                unFollowButton
+            } else if session.type == .fixed || session.type == .following && !isFollowing {
```

Comment:
I don't get the `session.type == .following && !isFollowing` <-- doesn't .following mean that we're following this session?

Reply:
Wouldn't `session.type == .fixed && !isFollowing` be enough?

Reply:
again, you are right 😄